Koha/REST/V1/ERM/CounterFiles.pm

```diff
@@ -0,0 +1,246 @@
+package Koha::REST::V1::ERM::CounterFiles;
```

Comment:
Opinion regarding all api controllers: should we just implement the methods we need as we go instead of having all the methods listed initially?

Reply:
That's a good question.. I tend to go with adding them in one hit if they're just simple endpoints (i.e. mostly boilerplate).. but only implement as needed if they're a little more involved.